docs/automatic-release.md

```diff
@@ -58,4 +59,6 @@ jobs:
     uses: inpsyde/reusable-workflows/.github/workflows/automatic-release.yml@main
     with:
       NODE_VERSION: 16
+    secrets:
+      USER_TOKEN: ${{ secrets.MY_TOKEN_SECRET }}
```

Comment:
Let's use a more "professional` token name in the example. 😉
```suggestion
      USER_TOKEN: ${{ secrets.WRITE_TOKEN }}
```

Reply:
Is it only a write token or could we also inject a read or read-write token here? does that make any difference? I would suggest to name it just `TOKEN`.